server/config/questions.json

```diff
@@ -1,6 +1,43 @@
 [
     {
         "name": "Participant",
+        "text": [
```

Comment:
I agree that including text objects in the `questions` section would make a bit more sense. The reason I implemented it like this though is that some text nodes can be separate from questions (like the footer text denoted with `"for": "end"`. Also, this method makes parsing and the JSON schema simpler.